deployment/helm/published/planetary-computer-tiler/values.yaml

```diff
@@ -12,6 +12,7 @@ tiler:
     minDbConnections: "1"
     maxDbConnections: "1"
     webConcurrency: "1"
+    request_timeout: 30
```

Comment:
My intention is to have this process call extend to 30s and the gateway be set to ~32 seconds, to account for "untracked" time before the api actually starts trying to process the request, while still avoiding the error from frontdoor.